components/automate-chef-io/content/docs/iam-actions.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "IAM Actions"
+description = "Reference Actions for Roles"
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "authorization"
+    weight = 30
++++
+
+Reference the chart on this page when creating a *Role* to know which action grants access to what page in the browser.
+
+*IAM Action* lists the associated action or actions required to access that page in the browser. 
+Use `*` in these actions to give broad permissions to perform all associated actions such as get, list, create, delete, etc.
+Specify the action to restrict user access to the specific action.
+
+
+|  Task           | Browser Tab     | IAM Action       | API endpoint  | URL       |
+| --------------- | --------------- | ---------------- | ------------- | --------- |
+| View Events | Dashboards | event:* | /event_feed | https://{{< example_fqdn "automate" >}}/dashboards/event-feed |
+| View Service Group Data | Applications | applications:*  | /applications/service-groups | https://{{< example_fqdn "automate" >}}/applications/service-groups |
+| View Client Runs | Infrastructure | infra:nodes:*   | /cfgmgmt/nodes | https://{{< example_fqdn "automate" >}}/infrastructure/client-runs |
+| Manage Chef Servers | Infrastructure | infra:servers:* | <endpoint> | https://{{< example_fqdn "automate" >}}/infrastructure/chef-servers |
```

Comment:
Currently under a feature flag. API details not developed yet.

Reply:
weve got a product named manage, wonder if the use of this word might confuse folks?

Reply:
Fair point. Let's find a good synonym.

Reply:
The description for this page in the browser says "Manage Chef Servers with Chef Automate," but context is really important. Thank you @susanev for catching that. 

Would "View Chef Servers" work instead in the meantime? I'm basing this on how "Client Runs" has "View Client Runs" and Client Runs is listed with Chef Serveers in the same Infrastruture section.

Reply:
you cant edit anything for chef servers right now, so i think view works

Reply:
but my comment was about the use of manage across this page, not just here for chef servers

Reply:
We're exploring this more, but I'm not sure that we can give up the word 'manage'--which accurately denotes the API actions-- because of the (deprecated) existence of Chef Manage. Changing some to `add or edit X` and keeping `manage` on those things that have parity with Chef Manage is also confusing for users.